extensions/libraries/redcore/api/payment/document/document.php

```diff
@@ -96,6 +96,7 @@ public function render($cache = false, $params = array())
 
 		// Get the payment string from the buffer.
 		$content = $this->getBuffer();
+		$jsonEncodedContent = json_encode($content, JSON_UNESCAPED_SLASHES);
```

Comment:
because the current function returns just JSON format as I see here https://github.com/redCOMPONENT-COM/redCORE/pull/749/files#diff-1c3406dd45e639096f91319ff7baa5f2R121
I  didn't check deeper maybe you are right and need return different format related with $this->_mime. 
I just resolve the conflict for some already existed logic.
I think we can do it later if somebody will need another format in the future.